### ISSUE TITLE
Add EMAIL SIGNUP FORMS to the site in 2 areas

### ISSUE THREAD

**Issue by JonBrooksX** [jetlej/HE-6.0-Bugs#247]
## I want sign up forms added to the site in clearly visible areas.

**1) A "download button" and image should replace the quotes area. Note: we have _LeadPages_ that can help with this if needed. **

Remove quote area:

![replace quotes](https://cloud.githubusercontent.com/assets/13107838/13015991/112587ac-d1b4-11e5-8eda-ae909273cb6c.jpg)

Replace with ebook download button and image: 

![sign up form quotes mock](https://cloud.githubusercontent.com/assets/13107838/13016018/2a273b92-d1b4-11e5-8837-dba34de711e6.jpg)

**2) Add horizontal page break sections which show trending articles along with a sign up form and social buttons.**

Include these page breaks after a block of 6 smaller post thumbnails. 

![page break sign up and trending articles](https://cloud.githubusercontent.com/assets/13107838/13016243/65bb32d4-d1b5-11e5-8cd9-bbab18c0f93d.jpg)

----

@dpatil-gmcs  @MSchirp @lostboyevsky

**Comment by lostboyevsky** [comment]
@JonBrooksX do we want bottom-of-blog-post button/signup form for emails too?

**Comment by JonBrooksX** [comment]
@lostboyevsky 

I was going to but then I saw that @MSchirp mentioned putting an ad spot there. Right, Martijn?

**Comment by MSchirp** [comment]
@JonBrooksX Yeah, that was suggested to me by the guys from Sortable. I have another spot I will add the sign up for e-mails. It will be below the recommendation widget on the right. 

Regarding this issue: Can you add the image you want to use (in your example) here? 
Can you also finalise the text on the 'horizontal break page section'? Now it seems you copied lifehackers example, which is great, but for Deepak, you need to add that the button style is the same as the other ones on the front page, and that the text is different. Also specify what you want with those social buttons.

I will assign this as a priority to Deepak once we have the .pdf done and the lead magnet up and running.

**Comment by JonBrooksX** [comment]
@MSchirp Okay, the ebook title and therefore cover is still undecided so we can put that on hold just for the moment. 

**The "horizontal break page section": **

@dpatil-gmcs 

The social button icons should be Facebook, Twitter, and Instagram and taken from the same icon pack that @jetlej used for the nav bar and other parts of the site. 

- Instead of "Popular Posts", I want "Trending Articles". 
- Instead of "Don't Miss Any Tips", I want "Join the HighExistence Tribe"
- Instead of "Never miss an article. It's free.", I want "Get our latest and greatest articles via email."

Let me know if there's anything else I need to add or modify @MSchirp

**Comment by dpatil-gmcs** [comment]
@JonBrooksX @MSchirp  Please provide the artifacts for the task 1.

**Comment by MSchirp** [comment]
Task 1 is postponed for now.

Task 2 should be implemented in the front page in the indicated spots (above the 300x600 ads), as well in the following spot:

Between the recommended widget and the new bottom 300x250 ad:

![screen shot 2016-03-02 at 16 24 45](https://cloud.githubusercontent.com/assets/13124056/13464869/581865ec-e093-11e5-9c33-222ea40491b5.png)

Now, for the e-mail form we use MailCHimp. I've already invited you to log in there. 

Please follow this link for instructions, we want them to sign up to Top Posts Digest, and automatically get into the 'Daily' Category (As Opposed to Weekly or Monthly). 

https://us9.admin.mailchimp.com/lists/integration/embeddedcode?id=417641

Find CSS customisation here: http://kb.mailchimp.com/lists/signup-forms/css-hooks-for-customizing-forms?&_ga=1.247268934.356490148.1452787187

The OK button should be have rounded colors and be purple, the same as the other buttons on the homepage.

**Comment by dpatil-gmcs** [comment]
@MSchirp  Which email did you use to invite me to MailChimp?
FYI I have implemented the Trending Articles Section and subscribe form on both the ome page and sidebar at the location indicated by you.
I am still working for the social media links, and I just wanted to confirm the URLs of the facebook, twitter and instagram page.
If we are providing the link to the social media blog/page just to follow or like the page, I suggest you to use the social media badges, which will allow us to embed follow/Like button in out site it self.

I have attached the screenshots of both the home page and side bar. 
Please have a look and let us know your views.

![homepage](https://cloud.githubusercontent.com/assets/5171153/13497236/4dda6854-e17a-11e5-968c-cb9eb0ad3c54.png)
![sidebar](https://cloud.githubusercontent.com/assets/5171153/13497240/5116ba18-e17a-11e5-8cdc-8e7a01586230.png)

**Comment by dpatil-gmcs** [comment]
@MSchirp  We had look into the social icons implementation, and found that the social icons implemented previously are through the svg file. It doesn't include code for instagram, Please provide us the svg file for instagram, so that we can add instagram icon. or Please provide us the steps to include the instagram icon in the svg file.

**Comment by dpatil-gmcs** [comment]
@MSchirp  In order to configure our subscription form with the mail chimp account, we will need mail chimp API key, However we are unable to generate one as we do not have permission to generate. Please provide us with the API key.
The steps to generate API key are:
1. Click your profile name to expand the Account Panel, and choose Account.
2. Click the Extras drop-down menu and choose API keys.
3. Copy an existing API key or click the Create A Key button.
4. Name your key descriptively, so you know what application uses that key.

You can also find the above steps at https://us9.admin.mailchimp.com/automations/ under Find or Generate API key section.

**Comment by MSchirp** [comment]
@dpatil-gmcs Sent it on Slack.

**Comment by JonBrooksX** [comment]
This is the instagram svg file: https://www.dropbox.com/s/j5g9id9567n14fs/instagram.svg?dl=0

**Regarding the design:**

Recommendations sidebar: 

The social icons should not be all blue, and they should not be vertical. They should be larger than they are now, horizontal, and black. 

I think we should encase it in some kind of frame or box too. At the moment it just looks like the form elements are floating. 

The "subscribe button" looks good but the spacing isn't right. It shouldn't be underneath the form and to the right. I would like to see the "subscribe button" on the same line as the email address form. To do this you will have to make the "subscribe button" and the form slightly smaller. 

Changes:

- larger social icons
- black social icons 
- horizontal social icons
- "subscribe button" on the same line as the email address form
- put a frame around the box

I'd like the heading text in the recommendations sidebar to say ## "Never miss an article!" and the body text to say **"Let us explode your inbox with insights"**

Homepage subscription form:

Same changes as recommendation form:

- larger social icons
- black social icons 
- horizontal social icons
- "subscribe button" on the same line as the email address form
- put a frame around the box

However, the text should be different. 

I'd like the heading text in the homepage email form to say ## "Join the Revolution!" and the body text to say **"Grow with likeminded people in the HighExistence community"**

Please let me know if any of this doesn't make sense.

**Comment by dpatil-gmcs** [comment]
@JonBrooksX @MSchirp  I have made some changes according to your suggestions, However I am still working on UI and FYI the plugin we are using currently doesn't allow us to make two seperate forms for home page and sidebar, I am looking for some other plugin also. Please have a look at attached screenshots and let me know your views on it!
![homepage](https://cloud.githubusercontent.com/assets/5171153/13572542/ceed5ffe-e4a0-11e5-928f-6ec8f3fe0b64.png)
![sidebar](https://cloud.githubusercontent.com/assets/5171153/13572544/d0d83636-e4a0-11e5-87d1-bc011102d0e4.png)

**Comment by dpatil-gmcs** [comment]
@JonBrooksX @MSchirp  When we try to add instagram svg to the project some how the facebook and twitter icons also get disappeared, It seems that GRUNT is creating some file and something is missing in icons files.

Can you guys please provide us with the steps to add an svg to the project?

**Comment by MSchirp** [comment]
@jetlej Do you know this?

@dpatil-gmcs I think what Jon meant was that the icons should be encased in a box. Not the whole widget itself.

@JonBrooksX I am really not a fan of the trending widget. I thought we we're going to post a small box above the 300x600 ads on the front page and on the sidebar. Thats it. How it currently looks seems to destroy the initial design.

**Comment by JonBrooksX** [comment]
@MSchirp I agree. The idea of using page breaks with social buttons and a signup button is good though. Let's think of ways to make it work without destroying the current design. I will think on this and get back to you with a solution(s).

**Comment by dpatil-gmcs** [comment]
@MSchirp  So you want us to remove the widget border and apply it only to the social icons?
Each social Icon in different box?

**Comment by MSchirp** [comment]
@dpatil-gmcs Please wait until @JonBrooksX has suggested a way forward.

**Comment by dpatil-gmcs** [comment]
@MSchirp  Sure.

**Comment by JonBrooksX** [comment]
I now think it causes too many problems having BOTH the social buttons and the subscription box together. So I'd like to alternate them on page breaks. This is a mock up closer to what I'd like. Take note of the spacing and colors. The "trending articles" do not have blue text anymore either, stick with black or grey.

Social button form:

![page break social buttons](https://cloud.githubusercontent.com/assets/13107838/13726265/93fccfd6-e8b7-11e5-9b41-033ea435d8a0.png)

Subscription form:

![Uploading page break social buttons.png…]()

**Comment by dpatil-gmcs** [comment]
@JonBrooksX @MSchirp  We have made the changes according to the Jon's Suggestions.
Please have a look and let us know your views on it:

First Line Break, contains subscription box:
![first_linebreak](https://cloud.githubusercontent.com/assets/5171153/13744765/114b6f02-ea10-11e5-8739-b9478c5803c9.png)

Second Line break contains social button box:
![second_linebreak](https://cloud.githubusercontent.com/assets/5171153/13744787/2aa12424-ea10-11e5-9b89-e0b9f73654b4.png)

We have repeated the twitter image for now, Just because we are still waiting for your response on steps to add the instagram svg file to the project, and to have a look and feel of all three social buttons. Please provide us the steps to add instagram svg to the project so that we can add instagram image.

Please let us know about sidebar, Which of the two forms do you want to add to sidebar?
Also, We need the urls of the faceook, twitter and Instagram pages to link to the social icons.

Let us know your views on it!

**Comment by JonBrooksX** [comment]
@dpatil-gmcs 

Please make them as identical to my mock-up image as possible. See image below for instructions:

![post break](https://cloud.githubusercontent.com/assets/13107838/13797748/d6857310-eb07-11e5-9d8a-d082915d7a49.png)

- Add the a green-blue-purple color gradient to the STROKE on the box. 
- Reduce the height of the "subscribe" button. It should have the same height value as the form. 
- Move "trending articles" higher (see above image). 
- Change blue text on the trending articles to dark gray. 

Please let me know if you have any questions. Refer to my above images for details on how I want it to look.

**Comment by dpatil-gmcs** [comment]
@MSchirp @JonBrooksX 

We have made the changes according to the @JonBrooksX  Suggestions, Please have a look and let us know your views on it.
![changes_subscribe](https://cloud.githubusercontent.com/assets/5171153/13812778/0d06c91a-eba4-11e5-9fd1-09cae90c3504.png)

Please let us know which form do you want in sidebar, and also we are awaiting for the steps to add svg file to the project.

**Comment by MSchirp** [comment]
@dpatil-gmcs Since there is only space for 1 box in the sidebar, we actually need to have 1 box, not 2. Your latest box looks great, well done! 

- Could you add the svg's to the latest box, just below the text 'Join the Revolution!' and love the e-mail address+subscribe button?
- Our accounts are:
https://www.facebook.com/HighExistence/
https://twitter.com/highexistence
https://www.instagram.com/highexistence/

Im not sure what you're asking regarding the svg file. If you can't find the answer online, please ask @jetlej on Slack.

**Comment by JonBrooksX** [comment]
@MSchirp it's taken a while to get to this design, and it will look cramped with both the social buttons AND email subscribe form. 

I suggest we alternate the two boxes every page break.

**Comment by dpatil-gmcs** [comment]
@MSchirp  Mail chimp plugin, provides us with an otion to design only one form. So we won't be able to show different form the sidebar. 
I think we can  add the forms with the social icons just after the top AD and  the subscribe box before bottom AD.
Please let me know your thoughts on it.

Yes, I am using Sajari for Trending Articles.

**Comment by MSchirp** [comment]
@dpatil-gmcs Sorry for the confusion. I want to just have 1 box, the same one on the sidebar as on the front page. The box needs to include both the e-mail form as the social buttons. 

So 1 box total in this order:

1. Join the revolution
2. social media icons
3. e-mail sign up form
4. Grow with the... text

---

This is what Jordan had to say regarding svg:

> Ideally it would be from this icon library
> http://ionicons.com/
> to match the size and style of the others
> then Deepak just needs to drop the .svg into the themes/hethen/images/icons folder and it’s then available for use in the theme:

**Comment by dpatil-gmcs** [comment]
@MSchirp 
We have implemented the social media icons in the side bar subscribe form,
Following are the screen shots :

1. Home page subscribe box:
![front_page_subscribe](https://cloud.githubusercontent.com/assets/5171153/13954273/18660eda-f066-11e5-9dda-c1b3ab606743.png)

2. Home Page social icons box:
![front_page_social](https://cloud.githubusercontent.com/assets/5171153/13954296/28b039be-f066-11e5-9b64-6b7235042780.png)

3. Sidebar Subscribe box:
![sidebar_subscribe](https://cloud.githubusercontent.com/assets/5171153/13954313/37bc4646-f066-11e5-9d68-6c59a82600a8.png)

Regarding the steps given by jetlej,
a. We are unable to use image when we add instagram svg without Grunt.
b. When we add the instagram svg using Grunt, sicon CSS files gets updated and even the other two icons(facebook and twitter) disappears.

Please let us know your thoughts on it!

**Comment by MSchirp** [comment]
Looks great, good work Deepak!

In the sidebar subscribe box, can you increase the space between the social icons and the e-mail sub box? So the icons are exactly in the middle of the 'Join the Revolution' text and the e-mail box? Thanks!

Will ask Jordan, will get back to you.

**Comment by dpatil-gmcs** [comment]
@MSchirp  Sure, we have increased the bottom distance so that the social icons appear to the center.

Jordan has tagged me on slack, I will explain him the situation on slack.

**Comment by JonBrooksX** [comment]
@dpatil-gmcs Looks great! One thing: the stroke gradient is now a blue-green, and those shades of blue and green aren't the exact HE colors. Use these three colors in the stroke gradient:

Purple: #7a5bed

Blue: #46b8e9

Green: #73c616

**Comment by dpatil-gmcs** [comment]
@MSchirp   We have increased the distance between the social icons and email box. Now the social icons looks in the middle.
Also, We are discussing with Jetlej on slack, about adding the instagram icon to the project.

**Comment by MSchirp** [comment]
Awesome, let me know when the instagram icon is added and the gradient is updated as per @JonBrooksX 's comment above.

**Comment by dpatil-gmcs** [comment]
@MSchirp  As per the talk with @jetlej  on slack we have committed the code to the TEST site under subscribe-form branch. So that he can see the problem we are facing in adding the instagram icon and help us.

**Comment by JonBrooksX** [comment]
Can you confirm that you saw my color request @dpatil-gmcs. I am still not seeing purple on the test site.

**Comment by dpatil-gmcs** [comment]
@MSchirp  @JonBrooksX  On the test site the code is now visible, Please have a look and let us know your views on it.

In the sidebar the instagram icon and the social icon CSS is not visible properly, I am investigating the code, and looking at it.

But as per @jetlej  The reason could be cachine plugin and he has asked me to deactivate the W3TC plugin from the TEST site, So I am moving forward with it.

**Comment by JonBrooksX** [comment]
@dpatil-gmcs I'm still not seeing any purple on the stroke.

The Instagram icon does not look like it's the right icon. I would rather just have Facebook and Twitter and NO Instagram if we can't solve the icon problem.

**Comment by MSchirp** [comment]
@dpatil-gmcs I am not seeing it on the sidebar. Do you @JonBrooksX ?

**Comment by dpatil-gmcs** [comment]
@MSchirp @JonBrooksX  Just an FYI The branch on the TEST site is changed to PRODUCTION, in order to test #34 so the changes wont be visible now.

**Comment by dpatil-gmcs** [comment]
@MSchirp @JonBrooksX I have implemented the purple color on my local site,
Please have a look at the color in the screenshot attached below and let me know your views on it.

![subscribe-form](https://cloud.githubusercontent.com/assets/5171153/14606303/43d7b886-059a-11e6-82aa-cef2be941eb0.png)

**Comment by JonBrooksX** [comment]
Hey @dpatil-gmcs:

Not quite right, but you're almost there. 

At the moment there is pure green color on the left and top sides and a purple-blue gradient on the right and bottom sides. It's a harsh transition. I'd like to see a smooth gradient going from green to blue to purple. It doesn't matter if the gradient is diagonal, horizontal, or vertical. But it needs to be smooth, with no harsh transitions. 

Let me know if I'm not clear.

**Comment by dpatil-gmcs** [comment]
@JonBrooksX @MSchirp  We tried making the color transition bit smoother, Please have a look at the attached screenshot and let us know your views on it.

![smooth-gradient](https://cloud.githubusercontent.com/assets/5171153/14711540/4b05eeca-07f7-11e6-806d-c459eb1ba2c6.png)

**Comment by JonBrooksX** [comment]
It's better @dpatil-gmcs, but it still isn't right. There is too much green on the top and left side. 

Check out my example below for what I'm looking for:

![gradient stroke opt in form](https://cloud.githubusercontent.com/assets/13107838/14732288/6ef0111c-0883-11e6-9498-fad8be98052a.png)

**Comment by dpatil-gmcs** [comment]
@JonBrooksX  @MSchirp  Made the changes according to your requirement.
Please have a look at the screenshot attached, and let us know your views on it.

![updatedgradient](https://cloud.githubusercontent.com/assets/5171153/14743107/83dbe5da-08bd-11e6-9874-05ae460cf735.png)

**Comment by MSchirp** [comment]
@dpatil-gmcs 

Please revisit the comments I made earlier in this issue:

There needs to be 1 box only. The same one for the sidebar on blog posts AS on the front page.
That one box needs to include both the social icons AS the e-mail form.
The trending articles should be removed. We just need the box.
The box on the sidebar of blog posts will need to be placed between the recommended articles and the bottom ad.
The box on the front page should be placed above the 300 x 600 ad. Please note that there is white space under these ads, so the ad should move down, and the box should be placed above it. (Perfectly centered)

Also note that I can't confirm changes on the front page of TEST, since it is still showing the broken front page. See hi existence.com FYI.

I hope this makes sense, hit me up on Slack if you have any questions.

**Comment by dpatil-gmcs** [comment]
@MSchirp @JonBrooksX 
Made changes according to your requirements, the code is committed to the test site, in a new branch.
You are not able to see the form at the test site because the PROD branch is main branch on test site.

Here are the screenshots of the home page and Sidebar after making changes:

1. Home Page:
![newhomepage](https://cloud.githubusercontent.com/assets/5171153/14785827/20b05a36-0b1a-11e6-9709-420da994ec61.png)


2. Sidebar:
![sidebar](https://cloud.githubusercontent.com/assets/5171153/14785850/32ff773a-0b1a-11e6-82ba-ceb356b4d196.png)


Please have a look and let us know your views on it.

**Comment by MSchirp** [comment]
@dpatil-gmcs This looks good! 

Can you confirm the subscribe e-mail directs to the 'Top Posts Digest' list, segment 'daily'?

If yes, please push it live and let me know on slack the e-mail sign up form is live!

**Comment by dpatil-gmcs** [comment]
@MSchirp  Yes We have an option to select the email campaign, We can select 'Top Posts Digests' list.

About pushin it to LIVE, The deployment is dependent on #34 so we will need a sign-off for #34 issue, before we deploy it to LIVE.

**Comment by MSchirp** [comment]
@dpatil-gmcs 

Its important this gets deployed ASAP, and I am worried #34 will take another few weeks. So please remove the dependency and implement it life.

Can you confirm it will add people to the 'daily segment' in the 'top posts digest' list?

**Comment by lostboyevsky** [comment]
@dpatil-gmcs, a couple more things to change before pushing this live:

1. Please add another icon for Instagram in between the FB and Twitter icons and link to our Insta account: https://www.instagram.com/highexistence/

2. Center the "Your email address" text in the text box, or at least move it to the right by ~10px. It's too close to the far left side.

3. Change "Join the Revolution!" to "Join the Movement!"

4. Change "Grow with likeminded..." to "Grow with like-minded..." (simply add a hyphen between "like" and "minded")

**Comment by dpatil-gmcs** [comment]
@lostboyevsky  We are unable to add the instagram icon to the Site, The Icon is not present in the project and adding the icon creates problems.
Please read the abpve thread.
We have made the other changes, Please have a look at attached image and let me know your views on it.

![updatedsubscribeform](https://cloud.githubusercontent.com/assets/5171153/15073748/ecdb5ec0-13b8-11e6-927d-da657ec1ea58.png)

**Comment by MSchirp** [comment]
@dpatil-gmcs Good enough for now, please push live!

**Comment by MSchirp** [comment]
Can you confirm it will add people to the 'daily segment' in the 'top posts digest' list?

**Comment by dpatil-gmcs** [comment]
@MSchirp  There is an option to select the LIST in the form. and about Daily Segment, There is no such option to select the segment/frequency. I think it will subscribe the user to all the available segments.
i.e. Daily, Weekly and Monthly.

Please let us know your thoughts on this, So that we can deploy it to the LIVE site.

**Comment by MSchirp** [comment]
Alright, thats fine. Please push to live, thanks!

**Comment by JonBrooksX** [comment]
Nice, looks good.

**Comment by dpatil-gmcs** [comment]
@MSchirp  @JonBrooksX  We have deployed the code to the LIVE site, We found some bugs in the previous implementation, Like the background image gets streched and the logo went bit upwards, We have also fixed those issues.
Please have a look and give us a sign-off so that we can close the issue.

**Comment by MSchirp** [comment]
@dpatil-gmcs Did you test actually signing up?

It doesn't work, please fix ASAP:

![screen shot 2016-05-11 at 15 27 00](https://cloud.githubusercontent.com/assets/13124056/15182469/e7beef92-178c-11e6-8842-b12ede580ad2.png)

Also, the button should be AJAX - so it shouldn't reload the whole page.

Also, please center this text: "Grow with like-minded people in the HighExistence community."

**Comment by dpatil-gmcs** [comment]
@MSchirp  It works, when I try to subscribe. Please check if the email you entered is correct!

![form](https://cloud.githubusercontent.com/assets/5171153/15182876/403833da-17ac-11e6-853f-4fbc649c66dd.png)

About AJAX, We will need to research about it, as we cannot find any option in the plugin.

By centering the "Grow with ...." text do you want me to make the text in the center of the text-box and Subscribe button?

**Comment by MSchirp** [comment]
@dpatil-gmcs It seems that specific e-mail doesn't work, weird, it is a valid e-mail and entered correctly. 

On AJAX for the button: let me know if you can find a solution!

With centering the text, it is now left aligned. Can you make it center aligned? And reduce the text to "Grow with like-minded people!"

====

**BUG**

The front page is messed up: there is a huge white space suddenly between the different articles, see screenshot: please make sure the space stays the same as it was.

![screen shot 2016-05-11 at 16 06 06](https://cloud.githubusercontent.com/assets/13124056/15183734/b4f9c982-1792-11e6-92a4-04f956ba6f7d.png)

**Comment by MSchirp** [comment]
@dpatil-gmcs Right now, the e-mail frequency of the new subscribers is set to 'weekly'. Can you change this to 'daily'? Thanks!

**Comment by dpatil-gmcs** [comment]
@MSchirp  We have solved the bug you pointed out above and checked in the code. We have also made the text center aligned.
Please have a look.

Yes sure we will check and change the frequency to Daily, I think the frequency must be changed from  mailchimp account. I will check it out.

**Comment by dpatil-gmcs** [comment]
@MSchirp  When I go th the plugin settings page, I find that there is a 'Edit listing' button under the 'Top Posts Digest' listing, On clicking the link it takes me to the mailchimp site where the message "Apologies, but you do not have permission to view that page. " is displayed.
Can you please change the settings to Daily or provide me the credentials which has the permissions to view and edit the page.
The link can be found under the 'Mailchimp for wp' tab in the admin section.

Please have a look at screenshot:
![mailchimpsettings](https://cloud.githubusercontent.com/assets/5171153/15214782/aff6f1e6-186c-11e6-95fd-a797f5271ffb.png)

**Comment by MSchirp** [comment]
See slack

**Comment by dpatil-gmcs** [comment]
@MSchirp  We have researched more about the ajax subscribe option in the plugin, and we found that the plugin supports the subscribe option via ajax only in the Premium version where as we are using the free version of the plugin.

Please let us know your thoughts on it!

**Comment by MSchirp** [comment]
Then we will get the premium version, Ill let you know :)

**Comment by dpatil-gmcs** [comment]
@MSchirp  We have installed the premium version of the plugin, Now the ajax subscribe seems to be working. Please have a look and let us know your thoughts on it!

**Comment by MSchirp** [comment]
Awesome, signing off!

**Comment by MSchirp** [comment]
(confirmed its working)
